---
std_msgs/Bool inRoom
